include/deal.II/base/polynomial_space.h

```diff
@@ -193,28 +194,24 @@ class PolynomialSpace
    * class. Here, if <tt>N</tt> is the number of one-dimensional polynomials
    * given, then the result of this function is <i>N</i> in 1d,
    * <i>N(N+1)/2</i> in 2d, and <i>N(N+1)(N+2)/6</i> in 3d.
+   *
+   * @warning The argument `n` is not the maximal degree, but the number of
+   * onedimensional polynomials, thus the degree plus one.
```

Comment:
```suggestion
   * one-dimensional polynomials, thus the degree plus one.
```

Reply:
But I don't think the description is right. You might have the 1d basis `{1,x^2,x^4}`, in which case the relationship between number of polynomials and degree is lost.

Reply:
Good point. The original documentation was spread around in an odd way. Git labeled my insertions and deletions here in an odd way as well, but essentially the documentation from `n()` and `n_polynomials()` both said different things that seemed important to keep, so I copy-pasted both original pieces of documentation together there.

I'll correct the typo, but I'm not sure how that basis concern should be dealt with.

Reply:
Just remove the comment altogether -- it's wrong, after all :-)

Reply:
(Remove the last half-sentence, that is. The remainder of the sentence is correct.)

If you want to improve the readability of the code, you could also just rename the argument to `n_1d_polynomials` to make its intent clear.)

Reply:
Okay I've deleted that comment :)